api/storage_service.cc

```diff
@@ -1423,13 +1423,20 @@ void set_snapshot(http_context& ctx, routes& r, sharded<db::snapshot_ctl>& snap_
         });
     });
 
-    ss::scrub.set(r, wrap_ks_cf(ctx, [&snap_ctl] (http_context& ctx, std::unique_ptr<request> req, sstring keyspace, std::vector<sstring> column_families) {
+    register_path_routine(ctx, r, "keyspace_scrub", ss::scrub, {"skip_corrupted", "disable_snapshot"}, [&ctx, &snap_ctl](std::unordered_map<sstring, sstring>&& val) {
+        bool skip_corrupted = val["skip_corrupted"] == "true";
+        bool disable_snapshot = val["disable_snapshot"] == "true";
```

Comment:
Those should be cql bool parameters rather than strings.

Reply:
As of right now, the CALL statement only accepts string values for its parameters